Winapp3/Winapp3.ini

```diff
@@ -108,9 +108,15 @@ FileKey1=%ProgramFiles%\UltraISO\lang|*.*|REMOVESELF
 Section=Dangerous Long
 Detect=HKLM\Software\Microsoft
 Default=False
-Warning=This will delete most kinds of backup files with any software throughout the entire system, which will prevent recovery of some software.
+Warning=After using this, you may not be able to use some softwares recovery/restore abilities.
 FileKey1=%SystemDrive%|*.ABF;*.ABK;*.ASD;*.ATE;*.ATI;*.BAK*;*.BCK;*.BCM;*.BDB;*.BKF;*.BKP;*.CBK;*.CRDS;*.CRYPT*;*.DA0;*.FBK;*.FUL;*.MBF;*.MIG;*.OLD;*.ONEPKG;*.ORI;*.QIC;*.RBF;*.SBB;*.TMP;*.TRN;*.VPCBACKUP;*.WBCAT;*.WBK;*.XLK|RECURSE
 
+[Desktop.ini Remover *]
+Section=Dangerous Long

```

Comment:
This entry needs a warning like "This possibly will break the function of shell folders. Do not use on non-English systems.".

Reply:
Added it.

Reply:
Thanks!